docs/version0.38.md

```diff
@@ -46,6 +47,12 @@ This option controls the handling of the `SIGUSR2` signal by the VM signal handl
 
 For more information, see [`-XX:[+|-]HandleSIGUSR2`](xxhandlesigusr2.md).
 
+### Technical preview of CRIU feature
+
+A technical preview of CRIU feature is included in this release. It is currently available for OpenJDK 11 and later running on Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only.
```

Comment:
Shouldn't start a sentence with "It", it's ambiguous. The word "currently" suggests that the support will be extended to other Java releases and operating systems, which is fine if that's the case. Given that CRIU is a Linux tool, it probably won't be available on other operating systems though? And is there a possibility of porting to Java 8 later? The word "currently" might need moving or removing. The trademark on "POWER LE" should be on POWER not LE, and we should expand LE. I suggest "POWER&reg; (Little Endian)"

Reply:
Done